src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.CodeAnalysis.CodeActions;
 using Microsoft.CodeAnalysis.CodeFixes.Suppression;
 using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp;
```

Comment:
I think that the current class should be shared between both C# and VB. It should not refer to any C# specific classes.

Reply:
thanks, moved to UpgradeProjectTests.cs